OvaryModelGenerator/OvaryModelGenerator.h

```diff
@@ -8,20 +8,24 @@
 class OvaryModelGenerator
 {
     public:
-        int nslices, nrot;
-        double d, h, w;
-        std::string outdirstr;
-
-        OvaryModelGenerator(int slice, int rot, double de, double he, double wi, std::string outdir);
+        OvaryModelGenerator(int slice, int rot, double de, double he, double wi);
         ~OvaryModelGenerator();
-        OvaryModelGenerator(const OvaryModelGenerator &other);
+        OvaryModelGenerator(const OvaryModelGenerator &other) = delete;
```

Comment:
It's good practice to always explicitly define copy and assignment constructors. But it's not always necessary to enable them. We can also define them as "deleted". A general safe practice is always delete them unless they are absolutely necessary. Here, for a filter like generator class, we don't need them to be copied or assigned. So we delete them for now.